src/pages/_docs/buttons.md

```diff
@@ -125,6 +125,18 @@ Replace the default modifier class with the `.btn-outline-*` class, if you want
 {% include example.html code=code wrapper="btn-list" centered=true %}
 
 
+## Another color button hover
+
+Add the `.hover-btn-*` class to your button to change the hover color if you want diffrent than basic. 
```

Comment:
add note that it only works for default buttons - outline and ghost is not supported